codalab/lib/bundle_store.py

```diff
@@ -422,5 +426,8 @@ def get_bundle_location(self, uuid):
         if storage_type == StorageType.AZURE_BLOB_STORAGE.value:
             file_name = "contents.tar.gz" if is_dir else "contents.gz"
             return f"azfs://{self._azure_blob_account_name}/bundles/{uuid}/{file_name}"
+        # if storage_type == StorageType.GCS_STORAGE.value:
+        #     file_name = "contents.tar.gz" if is_dir else "contents.gz"
```

Comment:
This is blocked on getting bundle locations / bundle store methods working, as that's the way we'll distinguish between Azure Blob and GCS in this method.